### DISCUSSION TITLE
计划本周日（7月23日）发布新版本

### DISCUSSION THREAD

**Discussion by LFTEC** [LFTEC/LearningFucker#27] [Announcements]
新版本将适配全新的万华学习，可实现必修课和选修课的学习和考试功能。

**Comment by HuiLiang1994** [comment]


**Reply by LFTEC** [reply]
激活码请查收

a2vbsEhNrs6Hn7InS5zH3XY3IrV9qm61GqMX6Nn/ZBwrEyDE6U0OXm16sEYfWov7zLwHtSiAFVRHTeflJOIRPok5XEl4pm3fDQYwpvcXib2eWfBhXeOCyZKotO3B8k7AbDy3SqK1tG9Y8gXUxHheKivTIb1kVLnZMn1FJvC32rB6ZHR1EN+OXisWbjGi+/FMxIBhbgLWYlrrl1qIRyShtu+rCp9wT+aIFTCRZS6Jo9HBMQIRC/RjwDu5RkkoP+oH24v10s12n51J5jOx0YgEzckFzDkx39pG9l0P1/4AiZZf5ubq89Fop/GQ7jeUOb38S2vCm1ulji9Fk2i8INzZOnafknccBZ4AenRJ+zlHEJsNaugkOBsSY7nX8aYxXdi1QL5UhCDZv1Sqys/k0JAQun3yorcfhT4TVu+YDgvYV7yJzcUMXen2bXT7Orud5rkG51JKWrjMHvKOFKNn0pUMjBikMhq/KowNAgraA5euQUw5ocYv7BSNzoBYTSdy4X/9v6TqErKwmOzsWWsrtoCkaxhYfAwV8AGes1pBfk1aZquIDvOL/78g2EcW2zgBaMM2Czap6nAzfAPcAPwLvi/vasKtVMKSC80BRiiRFlAiUWPlrgu82EzszAFid8IIEZM9TgbinvB0WoGMgKy4uYFM1G7iziMiGOxYbskCAimfBHUJXf+ITikUV+AqlJU9AMuSzeS/hnnfDvlUfklUWhfcZtRAFS4tPpq4uLsU5gVoQXcVkU/sx0tSEuCbkeXxCAzOi+TQoDTetEaTaf6E3QO9OWe+XDW5h7s55zs8reB/jf2kgm2sqOTV24Jb+I5MgvXMbcJCfv0wHpLAWpoQb8xyd2B3fh7x7BaK6eCIydQjPrPiwNmkjb9djNa8cTmSW+WrDuKYvjQKZTWBwGkVJaKq2LPcDYfr8BWtqUk3IRjXsrPlWWik19+d+UEW6YfMlp9dGoIS5Rvhp3nA8Es+FwQx+1ZH2W9FrZ+OUxEKzOVrX4vVgBw49kU55XDP5uO+8F3aZa19Eb5FNn7sG3l6WZpzzVsKn8zoa3q9JuLquP3Trx8Sw7+YqJuWwkOkINCZhuXZJdVIrNDDs542sWDAtOssBTfRWpjwaGEU66xbjkDDJ+m8bs3yX03iZNIEz8m9XerQ+L3WBYK4no9jeGSHU9dqKeCUve5Dy91XpJ7kxS2muFPU3ed5DEKl5t1/eAQnu/CaEf4OMt/UPamKjeuBSSgS8ktnIjxFebGUZHEcAuMo4nIRVsWp+xpw42pM7NOi+dRXnJHNepu++WPSBKjB8tos5lZHEAwIdrmwhV1Y8c/qqvcYR9LNux/OCdBgtfBWPbTVKpHLbFzvUZ7Ibpe5PIQEsQ==

**Comment by TaixuJiang** [comment]
YaezUZttmVkTkWRYJEe3FBYlo7UYh5/27MbmuNdU+oGzK3vRkb2Q9IuX3aXU2u1JbDA
iJdUdnve+sb1g7kkmdmw+8Q0nAboJRWSrZh/NdhffUfo+6mO1b0t9E0wmLTawXfMDed
78GYG7M1TMGyk3otoe3q6Cj9sayMxnGZD1smvdmtbVRbwRb/xT+ekwBWUDZ1HD5l7fx
ZT4d5cuCRCxrqFuyWDn4tXU8+hppFcG9O1UnBushLbr2MZVhZF50FDUaTIB8MKff+PN
b1W5Dum/pHswbUoKeTyUfkQm8ZV433HDFBddrbJJ+F4RAJaBMwRz4//AApwAhACEeny
FSxH4ecnjGvnsAR+lNcqU5wbkFbKudmAZy7M+jHiyYizKQ1nYsk7ffE1lyFNQzRU0Rb
b0e2hNIHDsLHk7HKuUUDufjPS10BTjiaOuBSayCUOZ0EAPDpmJf1FGAUe9WoyYa4vsu
Ma+WZM/tinUetWnU5+7StJDJQJUIbWwRiUk/gSzEC6Mua02NF+sL0K6aFpOOyeAIqjB
jFsGJjwDF3fWMstD6rrnFXaMLMB2YsgAIp/IIK1U+LdRMlNC7zMFJDClD53wAhg59oC
LwN8/iccdxxhqw+P6cnmmGYtDci4Mqz0dL5suR6kg4NgG97xHqUOsBN637zQ8g0p3jt
xxyFPALC3+SmmfKgPI2LrBmLi7uGtS4C2uakB8FA2zPyGC1qCD0EZ62G2wuUurBqpFP
f8F476K2oh3aKdLS/jknb9JHT9VnRw3jLT/G3wbWIZgGQg7/h0VJwNgvwfT6Fpam0ih
9J76Z2TqxtZ0/zRqV2TKHgrRtBfw4lLbgIkl+hYLG78IilohdciMx3pOxU/5eYRvdhW
XSO3D/nV6IdJ255fcXkvqh/P+IuiXynDBibktD0MIFzetNk2ui5vCwzuTSQTr9aVpq9
A60bDvBsgd/K/8St25HVzsw54ANwCYEQMBNaGT8Anlb7mTQ4FkjU9aZ2dDsKVP5DTp5
HobmwwulLcEmYLnikzXb1+LKu5sF2NS++oVjeGKS4CH1/sJMvWiPlx5sBUIBZkOZ43/
OJipqU2PW0TF3YKP92XhWX89MyHrbONZehXZfiSJ8y7jYXEBU78aJHogxFXFmV8t8/z
2wZr/wG4+fDre9RfXII7rqgfIbSCaxag3BvT0wynKWD45JKJwWapUmW6XXPm/7jZ+Mg
c6NZWjmXIyfJrq3qfgZPjJFZyzSJ17lQYKeAQ8jFnonfyhNFcX9g8s2NAG8nat9kL+R
PuuuPcmy//9C6On4HC58bemdjVeAaS6rFY/NNaW49ESWLzqh2sFVeApoEYU6zJhNrcq
TmQicC3V1HOyo1rjVy4g+QHD1Q==

**Reply by LFTEC** [reply]
QBFy0YE+iw4pK81rkZoRM51HAcSgAZ7SlNUaubB6YsD8DB6px7JrbjOOpwWb5D5vhLlWR5lo+VyJo+RNAmFzu4kyV0teZLNYk0LkYKRoB6DitWf4TNGfccEiKn2Y1YmGoTcgLqVfyiVQMyi1C6iHPzrrau/WVbLBRrU3HoRjEcFcVsUbvuHDPqKL7Tv8R8Snvkh3GUzkai5chwEn0q9tTShoSdzX+Guu8ge5fj2AZy60cWMYOQsJc/KNfg12/3QIK/YTjsrGZcZR+EOK8iAe3OMY0R8M1PtLLVdv5O2j67FxW53ynw3bqE307pmLNwHBMvNL9NcEJAuNcKANAfNXaAX6tzNMDtBlqR10SAO1ZuwaQMJSmzVTUhQ4fZwuzpj7nKyLgUAjEA2JEYkq9JSKlQFYrriUDKyLmQdnBjTlUasDqZ4K1Dhq1QORJ6NaUseGlgueFmK8RdgDID9m4eDhK98RI1aWLIoDpthpzETFnVi2FhSQx043ZIfPqJekq+ezWos1Bs112hLASM6vnUi/nWglN+g5RFFzAqnXVdCTFXkeLPV5LTJGwBVb4OHfbFxzC8g3IUjcqK/h+JxPCJ1Aw33hPA8dbLgCleBMWF/P0cqzSRvTHCBosLAEjGZs3jgNHb2cz+PwWlY8w9YPf3EfBA7PAnD6Z/bYjozLQTd3ETgAjqsfxnn1ac+2UPwhLGShs1eMcE6bPX16ZkJ870CewQDTQGLja9AvK/UAebY0hQwqEfRBq5Ox9ve1KwhwkfyLIG1KrxcgDuOjKEdcWVRDEZzqcAugc47LYfN85rFtriHR/8qSi1h5e9rnU7JEd2S6OL0YfByXwg7bJ3m1sGQdYbSq167lfRfQfiy6JrXHdYBN+3Pecknp19GRsBqa1tpjBvkH4aRzxD4Xoz3Neue/iASXwrMhszXlcQloK1cccLINxTXDxUKq5eA7yTsWCo8/q9HMjx4w49h6j5MKvsacwCELdOFrTfPiOR8d6CBbGknq1YiVPNiuo4w4ZLyVnuSaJ4RrUOYfd7rUJ6FiDEkHnfHR26L7cQlgTM3G9HTUL93SRyzaZUQVx7kXQtbRBNxvTPDJLCb2mXat/hsfhIMOXl4JL+7xD01rhc4dwun6UTB8KgSOh/hrTs94V8E+FQQ6suygpCfr+3oq9Ov0ZD5Rq8lXLLNPSbuIsLZjew9iv20C+B2nW+pLAptXSdlSDn0VQBNeg1RelmxYL1mFQ3Zw8YnQKuUoS0wxdP/nkBbAvWL861epAy6vB9ObAYqNU046jg8OfNLYEQiN2Z2liDXut2cDH25o9ravBuVlxt0j83q9b+Lf8OstGSdsfPj8uh6e+fUIdvPdfuY5zFfn/omhYw==

**Comment by wynegi** [comment]
i7FunyT0hzADYzhN+1b0nqyq3AA/Ym2PBcX8UAvaeTUjNtW5RgZgvxuFISYKypMOVsX
mqEscqzCnH1g2Rltd7l6QV8KzUCFKCO7rc0vibGa/6ys81jKKXHj5Y+XLDEfqxAVJJU
+97YDVS+xNxqVJJR6fTfW3tgoKPLM3Jd9V1G0peBp5okpqWQ7zUEq6eSzOwSZqfdgT+
AADIfihbY8FG54Uc4CX7J9xYSma1B91IS84N6cw/x2Q1Xf+DWxq1WBXy44rxGfuhp3S
06VbEsDXYsm8THpcwasz46wqgHWGXsDUbnYVP3qv+QBZVssecgCiUl1MVqEI0pwcQxF
KPLrqEQG6C2eTYG6LbYMymNR/BMD8Le/zz1sKJpSis0NrEepF2IAPpOL9cmHA+WY8LU
KPjTa4IdSqi9EKuOnxLdA6XGV6WQTjBmao8+Hlt++ddL8OJUXV1yv28RiPXVaNGqz6+
73XFUVkkCpn9VxXk07lBFiXgtHbuZ3kc96AQkHXDLW9NA21bFpbih58UK4IR/sOn5h+
6mHQcSvzjuWuYxkEoSLD4nTVv63ckPLUdl3kAcL/FQagzE5zAnKaqBGn61cRPPW8Q60
/GqglcLaiwVeYVDsx9JHMmBKBGQ19mOiHYbVYiGgQlgnk1Mxu2Ww+c707qJZkCTYhez
hRzY2uJMkrN8MuqUzjGQ3tmuQVH6XdqdYWpKo0BOX5Y5NuyjlokL/70Io2JNzvPwcFa
SbJ1eiNvf2yKuZsnybceLl6vrWTrhRPDFCYY8MqKNbAeBTUZErLw8tZu0WVVfW4swrS
psKILBshrv+A5r7MebmFworHWKY9zz8tKqEQQu2Nq7sacRgVKimhUKBqNxR5laJkBpe
uypVB28ZcKZolBo8bI5I2Dj4FDj7c3o9nN09wbNA296VwzozP2Dxc/4ExkThdm8YJZt
vgzW/og41xUwGpz0XTZM9DDq9jU0tbLgxv/lpsDran5V+j8whXWJERwv80YTeueQ1OG
hBCITnPEEO/6XU+AOGfY6Q1dX2ud9P5+3X4cafeduxnaot8JUDVq7N7QpZvZJjVyVIp
aFa9qjNkjrN6dPQebIjx8IOMpFESDT85LH9mtXns4aurDQBjimwsOD/PI8qp93bgoSR
JZuuvhyqfLjat8ogUEa2H2L/izj9mEfJfWK9IbjQk8DHLBQUpOupFQY9xsVX6iz6O8o
eL6Xzbs98lvKLkhSHnNn58CIHgaSEtH6Z+Lo1sC9pXV63/OcASoSpRPdXZvEkwtKra0
ld1AmecqvT/WykBhGGbhsfuck+6R8hnQxThFakDsQZNuneL4JZWlyuaLAtcBkqYhxkD
t6pxGgY/+GwNK4ie1PmTFN6MqQ==
你好，这是我的申请码

**Reply by LFTEC** [reply]
X+KKAzrbDpx6P2C6ybbdkySg9WCQ/xeFHbckzRoHBEK9CQYLtfayfHKPoSHyI6iiNimdYeokM/C4tmDcv5txFDZdBIchP+q42wLzAnNVjLq5cwvU/CrSGcre+T5fFTDd/vqUE3/sFoB2CsLtAUjY19aULsTOA1+elOGG3izJlTgzR/aHlVpL2SD6ndsk9Rv73fSIivFhoIf582cPQtPYEzTkTBncIoYfZcYvm1j7mKWOknloXn20OTa9L5cdHRWRMsi7HTCKSnxPDaNy9aKiIRlXQ/IbsNnOq5lrxERElL0s+xKb0PWo2GDMbkIeaatxmea/t3TlxxkVU+6fB3b1fG2udCR6JwgzHGl77zp4sQzFdzenK/BkZKPniIGjC8P7dyNPHpMHh5hw+rXpzb02DmZGsVLtiTIJOcioiY9awOl6UCVOYzJNwlC7e5T5+tQLESme75Ee1nGMgVj0RIPftvM2WB0CCcFzEmYFjjvhGeqiADSjYWz7fz5YEW51j5cG1dJ5FEwKyjNofZWflq5hREh9ejWfnX4ZnynLzm0UwRCrHRd3adMBp44i3plh7rawwxhJ21fk2XaU1YdVBSiyxy3UmrFjPihWQ4omjnVCBTW90yUAm03AymFQJvfWDcv7vZX/PfuuQwQHmEMOITbvEiLRxMlN8xddGG3zl+9v+AdNfghKXj+tq/sSqPZ7zWwJUpKRZHpSfJEsnJwHzanYsrw8Jt6S6mHhEQGc/lihcJ1fEVMATceMRJ35ex1yXwdCqm0pMcQa15gWjIUAQ4oZjmK7/KZVO0vcmMvUSHAU3YORvmKe/RZeCRtRpiHEW70EfPxci4WOzJIb2cZZ84lHMOYql+Lbe8p/WU4coZYUbhf0+NN2T5ses18XotyjWsTMUHpz2bG7+hFs5CMUD1dG4P4EJlNFAXqF5eMn126UtgpCDkz81bflUMWXNzWRTqjd63NzEilcm3NgWyGqdE+EBwKHNqFG1J+UnbE8DPvbqCRA2UHw0olGLBkFROZ9n1FLfqcb0fM4gSTz/ibE9LkbdAFhlA62j/9e0myrCnAk5dChh0hVXkWg1tHYEE8jI9NXHN9Cfb5QPMIMBk15ZZpVGChMQpZbM73uydssMGRVDb9nBz1e3RtdstvwU80oJ02a9XFRgmCe9w2SE9vw+aj9W7ZxywB8YKgLHq3zNwxaLkOWe8ohXaROzXloZIWtCK2Gpsin3qnDLK6u4bDHiWAHHHmTYBjz4nUkZ2Ifs0khswb+S24mwsmiQoflLxal5UGZpbkRn+SxaSfxxZZpvMMDEXyQSD6ulfO7Q5GjHnhps3GV1UU8nOX90pS15gbTCa4RGhDHd23QckYakjCk2w32OQ==

**Comment by CLOUDSEPHY** [comment]
rzJK+r+AKxOcg1znVvYDh5Ot/lnMYkCXgSv/tuKFxKld9Lbm6R70uGlkW7tOuW7+QDR
7Jvd00t2MIht3Q0Ez171pPxaJagQ/A7+A8FgXyiOQnbP4EiUJArYe7OcRpyNUHjRf2I
OEDmgC5xgXPVGRPBnBxyUe5lcvu3vVGcFHPVDv0G5OJISUY81rJgtQOs28rIoRBhBF5
Q0xzAVldxFlY0a2Ik7Vydn9GrbykJfDb71+k7OZ0H4DPBmQjQscyeFw8DZNoHttpiGs
Mehwfrjl1TYzGSj13FYrludg0C94PyG/FtoNQV2jnl8cPEkhYzNt7iPK6goJEOFAmly
IxQVOGuKq/MN38W3Zo+km/99bPZGuyIdAziOcZajUv+fHlxIuH37oRnYdLsZLfQziov
nu6VSr3gxyFqxo8Gg0Kh5tt5WFBumP0mIeLXOuf7ZjBDKayEudI5Lipqv7etlAAQ7bG
t0Ap+qWSgdXjksW28by8X07efMAIKb1iJDjDiSE8Uf8zCR3V/iC2mCzOn8HkTndbQlJ
lH3deBHbIdvoEfnLh/UmoljojTpRi7lxKmBaGul9vWxJnoARdCjQnswE8oXPqLHhuDC
c1XjxWqo+hKVmcHrz1qpcStYrMBRvi8YgLHeQWG3MCYJmm6kAJ6iHOgMtAEl2UzS1Ew
522MW2xLSFwLoqVbLXj75R5f99TYRyFjx7QLxZu0lrBwbVWQGQ4xoVwSJ+p58zGbsUP
xwhZ9BD4slYMZvR86xxhXI0VSKlwtHApeFdFgfcc4rKrFs5yG0QMdw07TCIZZ+1aQ42
y4X8jO/0UunSFMZlUtvUJiTpGmZYqNJuShyW2GyOGkLOlGdUIB91kcK31kwDyQMDrTj
cmA7iiPUDPJHpAzdCJiS3eoXzKy0AzzgevyvLDsLd1jnYSoXdlTng1T+NnUegPBQQQp
XoMjS4qMX5KAMMxBCVuNngBwO1nMoYLNOExcZVVjYlDiIgXxh5HiembW9e5/RwdD6vV
JuuCkFuK58kWlfVZ3GaDTx1SJ3h5dfi1SUzCE0JP0As0R7j++02UGW0ZzVVImIO1s2R
JUtcPoHHp7/dbLhJSlSax+xm7wmkC8FR5BKPIC4eKJ456+QPL4HLvKll4EhxY2QIKYr
6PY+AlN0tn0nc6q5+Pm5iRNxQ8584b7zrv2zWAW9aElOzIOEWBJ1KbYBdBhGgJ7B5L+
BJ8aSzWyCK5LOFeVwGDtL5cE8Z9jVK5WsyNUzeWfR9xfdu2WexL/KugCIZzQ3+GEHyf
2YMDXCdvmKxyVbJNZiDq54sWB6/NEaf8r+0y6iex7ZFLvRwm2Ijw9IBHF+KOPIt/QgQ
au5CfJXaJ1wz1xUH75rshoReug==

**Reply by LFTEC** [reply]
aL9gtNu6l7/WDAcqGBFRiEZHZ9NEhOy/YCS4asscLXxgVOk7N7GnikLbA50/BxoYoy706E2m/eK16NLJD8jcpzjyQWbB/Edn9VSj8JtbffRpBsHVOQ5MgIfrDxqw5oypTPozoIQfM+jspJ3s9qBZw0AvVXbPLHfJhQLXMYgLKJeXJjwHBrIBiRjfkRcwfzjDeJbOkG+ZlY+p9Pq2WNxxDa90JR8ILnDhgUC9K/wEqhP0jDWbv+h+DrIqUMbEKR8xhwab/giT8eCaN4k8+kEnIgjIYXicrRoPuqQ2aQY3OjwBwAmSdVk8pQOurHf0JcvE38IC2Ibxv4OieRjMYdo35IEQ/0Pt7sQFPAHyrh3waZtMNO84aM3B3vPz8sEeMgvbg+DI3IPizdgL5QIWVpL93UTMoonwMWeSCBOugtRmYqgPhCfi5wg0CqXPo6QfCBOelpAqxsG3C6yFyTCULRgcuTOqY586/SPH58MWQsUGSk72f2bHqACX/bkWye18fKYa2agOhqUo+ktjnnZAUiIkxXLB9X8ijdmWXkHEorviaAjgk0woVPehFDIr7iafskZqVJ4/klfUzdMIij9OBj8pduuFutxCdLtHkekPG28CQ828NAn6Kqf+JlUcYoK6tqbkyGVKeeKJGcikjJzjHNII0gv+JeZW78oEoreYfTKsGtYr175dIxHeiy9Ni0dd6SXtd3hxQXAwEvuzj4kCI1bDNATm6CD82T4AKgB7GdSYEtSMftWdVXifg1tQ8BkG0C+YLhg+QBiriDCwIGYGSv+FXLT9JZCqz22J1UHqZW/+sLLJHU3tU9wrOHGyMnf5jge8d7qABgRmJnKOcWGoBEqSt4xTHgLbdnRIkbWtI/M7xjyY+oHsrre7ULUKm3foWreVUmxWDdNeP5+gNRPWWbbbwAzKXsqv23IkKVnK58AeXvKE3zoS0ms0Rkp8sLimCiB6ipV2XJ3s4PJ5RRWz9efERTfe61gw8mStEY66qTlagdOk3Az5C5v3t1XABBMhniTvmbAZVDIFNN1tw5/Oj5U4maj+0mxyV3eoer9N9bxvd47VjHStKWG1ToMVLhRHDgJouuTWgtEqoUToibIcm2ng4R94dt5k5pNh1yIpzM4K8yUWCikrs4jnhBPzK+aADcePawc4ylAF3amO6qbs2Sp24yPrzg1ZaZAui2EsrGXVNJ0ZDWS4Aq+vBAJQbRcULLrfMiB6GUAgDdVwCAFew4jlKLdGdLOiTkvSbk2x8ad+woZwBJSjBK6i3BMvb0Ks98x/CrW1GjEKqEch1X/NGkXdToL2BzzY6dCyu7JB89uRQc4Wce3pOIH4YorCSF/+rPuhI8J/xlmwAkxU/0khTaGe8Q==

**Comment by ytAlbert** [comment]
dJDfdSEQdi03khSjHUSRR2+MdNdzMLYJuAZuxpsP6KUg229Xt7lbrlW1kFbpEdPqfDe
ye2TIjkLSS//pP9o/PSGzf5+PHM1OlsbwCsIK40lP16LpMvo7N5fTsDDmcRancemAxT
S4CSLWvPtmQw+fvWjpI43Gri4h+UFM+JOcrhzFBOhu8L3WfLW9RU85qdcqyCqwNf0yw
POEO19IfmmhxaFCDCKO74W5cHWlD4UbWdfVUt6i7tziPu4NABw6WWSD6caTt4eDNmjJ
da9XukvohSamYi+1naMzc2JjuDvpHyX26vHE2H59fCkFxO232P3ar5D2Wzdo/B6/2CU
CUO3xW791asytcdX8tRmm1RWpM5l4JuRfzPqsERt7ymseoLiYPEzKpE74uurZs0YRzu
D8FQVIK/kN9O/DpgRPV4+UH4I8j8arIeIg8ZFOg7n0rGN/Rz+vGvNRzeq/SKxqm1hG9
N3QpZBjXdjOFhYDLSPWg+isdQ9diOtDp4FBe+Q59DNh2ReSWvbtpQAJ6eM10JbEco1M
x0RQ/ZwJgs+EPV03oA+JBzi9F4zt98DmTuzF0GdEhptMAA5d8LjqTbHUUrHeS0lFPLa
DqYAEv0DTRaD/2QqkGbenfDhJXfAyUZdWUmJ/U4GFoaiPUoZ4lHVx6nlT6nvtj+CCrD
kgQa8aOtJc+cSDL1EfjxdavZFagFv6T7z5dRvCCts9oykBzjIr+RjW5Hi2eg2AQaP9Z
cODz6OEIbL2fmNSsOkZ9wBywu+ilxnZHcws1bwXWh6pFcxabAl9rJjtf5cL9kFaF946
20tr0/ECml1SyQJf42KpDlCvgQig8LGBZPe7wl5E489vV5ukQeT0OtKUeQOKXULzUfW
0qw9Om+ZO0N19iDF2pJkOP6u2dSnZV8D+Q1QNMeo7ZA8nBomCJXqcIfU7qT5Rlj5Y/7
6sWi0/cGYV70AGJ7pWfyxn+e6Y47yLWnzGSLRgPPinNT/8s5orr9mfE4d6CdBzzqgP9
h0l4VBNXUrF3G2LHlUUrEq/Rscu/bYhf//b7CqLuozJMTugS/skAzb0c8gZo5kYf1QW
aFHiELrsxefxgczNC3ip8aw5piAKhUqlkYyAjEo3ud7sEyE+S8iuvYork91r9jbdD2z
SpvlGKdLDot3nLuIJYVKHf+9eNsIOwmF+c2y5JZOR5R+nK+rnESr8SyyfyWYiZZTPVw
/N3CjwKbP7abeHBw0M7EFE6b0iBsLuQG35FJM0SoSn4BIH11dB4hFzN+TFyWEqAqeiP
vR30bf4JMX76OpewUvOex+B/KpG2OS8oCXeAUAI5PO2Ewenx8hbOd98r8U5nUI3nztk
g9GZyeWZF8240ADiyY7J3oi3uw==

**Reply by ytAlbert** [reply]
申请码

dJDfdSEQdi03khSjHUSRR2+MdNdzMLYJuAZuxpsP6KUg229Xt7lbrlW1kFbpEdPqfDe
ye2TIjkLSS//pP9o/PSGzf5+PHM1OlsbwCsIK40lP16LpMvo7N5fTsDDmcRancemAxT
S4CSLWvPtmQw+fvWjpI43Gri4h+UFM+JOcrhzFBOhu8L3WfLW9RU85qdcqyCqwNf0yw
POEO19IfmmhxaFCDCKO74W5cHWlD4UbWdfVUt6i7tziPu4NABw6WWSD6caTt4eDNmjJ
da9XukvohSamYi+1naMzc2JjuDvpHyX26vHE2H59fCkFxO232P3ar5D2Wzdo/B6/2CU
CUO3xW791asytcdX8tRmm1RWpM5l4JuRfzPqsERt7ymseoLiYPEzKpE74uurZs0YRzu
D8FQVIK/kN9O/DpgRPV4+UH4I8j8arIeIg8ZFOg7n0rGN/Rz+vGvNRzeq/SKxqm1hG9
N3QpZBjXdjOFhYDLSPWg+isdQ9diOtDp4FBe+Q59DNh2ReSWvbtpQAJ6eM10JbEco1M
x0RQ/ZwJgs+EPV03oA+JBzi9F4zt98DmTuzF0GdEhptMAA5d8LjqTbHUUrHeS0lFPLa
DqYAEv0DTRaD/2QqkGbenfDhJXfAyUZdWUmJ/U4GFoaiPUoZ4lHVx6nlT6nvtj+CCrD
kgQa8aOtJc+cSDL1EfjxdavZFagFv6T7z5dRvCCts9oykBzjIr+RjW5Hi2eg2AQaP9Z
cODz6OEIbL2fmNSsOkZ9wBywu+ilxnZHcws1bwXWh6pFcxabAl9rJjtf5cL9kFaF946
20tr0/ECml1SyQJf42KpDlCvgQig8LGBZPe7wl5E489vV5ukQeT0OtKUeQOKXULzUfW
0qw9Om+ZO0N19iDF2pJkOP6u2dSnZV8D+Q1QNMeo7ZA8nBomCJXqcIfU7qT5Rlj5Y/7
6sWi0/cGYV70AGJ7pWfyxn+e6Y47yLWnzGSLRgPPinNT/8s5orr9mfE4d6CdBzzqgP9
h0l4VBNXUrF3G2LHlUUrEq/Rscu/bYhf//b7CqLuozJMTugS/skAzb0c8gZo5kYf1QW
aFHiELrsxefxgczNC3ip8aw5piAKhUqlkYyAjEo3ud7sEyE+S8iuvYork91r9jbdD2z
SpvlGKdLDot3nLuIJYVKHf+9eNsIOwmF+c2y5JZOR5R+nK+rnESr8SyyfyWYiZZTPVw
/N3CjwKbP7abeHBw0M7EFE6b0iBsLuQG35FJM0SoSn4BIH11dB4hFzN+TFyWEqAqeiP
vR30bf4JMX76OpewUvOex+B/KpG2OS8oCXeAUAI5PO2Ewenx8hbOd98r8U5nUI3nztk
g9GZyeWZF8240ADiyY7J3oi3uw==

**Reply by LFTEC** [reply]
YAFs9AzNEzyjPILRDv7MZ3cZIrb+I3ltKIv4v6sz/HmmO2pdWrjxyrqhCWE/k+NQbVbjHNis7agy7Zck8Y3ImgM4Pksvx+/5++0lJHPEqTnmObnQx1d3KKOrbkeDfqRyFXYvzrmenJ9UdN2TGUCiyIbJEaFzmc8eIY/2JAEI7zfZoFWU90Fjb4dHe/eQxkHvkGJwZRo+Wurf/65C1w6MapP9K5ZLVRCQR6CJa0C2l7wNbbWFPPTwRVm1rdOimcBwSVOXyve36Risak3UWk/jJ76Mmebl6lwW7JUSDMC3qX1SYRYfkWdrelLDytrCsRDMTH9EHsieapQWMrF/MeCX73vjesPVWPN/3FwIO3dbxBVKiciCwCKltJjS3Gu19c0KatV2RpmDL3qr3+4qMtQSTg99nMx4omDJn0+0sC/OthDNHCpBEDhfiDsCqUrp4ljllGCBo57OvNU2k3yS5mEPOJuTJWiDvBlaLU2Krjk21th0O+q2OEqZfZs2lL7t1q5UZUYRkDs8reECnkL2lCgbVjUePDTDFZT/Sy0jKlPAA4KaQ9wM6P4fTejLINe/eX82/FGA7j7zCuhF6zb0aRp5ooPomPKHku8bvM/DZ0ktV25iYcXYbFRnVZBV7kPH/zhHBR0U62TByNPyZ713CMxq0l/eeW+JgmtBEvwXfI9nZf2BFbExJVxQxWcBADtBr5WQXeLM+32ExIrUKhOgRhpzguglxnlEvFStVMLFuyEc+P6eru8sbw7NLrkP+P2LjfExVB2wtbFK7Lt+N0Tt/XBOtjSgwzHZl8IGpUvyM6YBvctHk0zudr9+Rau3wk9vYbAMkryA2ucXuDPaq0dj0PoSnrDNhXiTadG275KWzO5IkQtN1+WUvyoQdgZOn9lfWEJM2KysoW8bl1SvTIdxlItFvdw/jO2rJ62CM7GwGSs/ukgDtOk4/EKgJW36NPnJoYbz1YFNswUGbsCk79BZQqR08tBss0PgMFFT4OWQ28ENPjECBR1MYEPnLJlUKo5NCuNgeljkEaKKn9KAr/NJT7TcAqPO5wfEQfXkYpoRPzvGa0Rt6sCrUbbZ9HndZgpV+Xql8nV5W/UeCvJX2MjzX1WHmG0XL6zdRt/E02K2nbi1TLUgwN4vLmePLni4XsYKZB4e8eTo2z+JivGw8hj4aVGwB4oIcoV6nr9OonjXpum24HJ5Ng8umgTPZwF3nTDC4Uvnx1uJXM4+5wWwbcDDwXBLhrQ/UZcja/Az/K6B56MsB/69whLAaV/IxyZxvf5/34FdrDUyrYFVO6fL3fEIVy+iKTyriNRtjDVVVxF8DyLsxjIv83OoeMuincKx7hrRYeq8FTi6HnVE7v3miQj2RlVv+w==

**Comment by LFTEC** [comment]
请大家申请激活码时，不要在此回帖。发一个新帖申请，否则我可能看不到
